src/main/java/net/snowflake/client/jdbc/cloud/storage/StorageObjectSummary.java

```diff
@@ -85,7 +85,7 @@ public static StorageObjectSummary createFromAzureListBlobItem(ListBlobItem list
       key = cloudBlob.getName();
       BlobProperties blobProperties = cloudBlob.getProperties();
       // the content md5 property is not always the actual md5 of the file. But for here, it's only
-      // used for skipping file on PUT command, hense is ok.
+      // used for skipping file on PUT command, hence is ok.
```

Comment:
Fwiw, I don't pick these corrections, Google Sheets makes the suggestions and as long as they seem right, I let it go wild. It's much better at this than I am.